js/interface.js

```diff
@@ -838,6 +1089,115 @@ $('.file-manager-wrapper')
   })
   .on('hide.bs.collapse', '.panel-collapse', function() {
     $(this).siblings('.panel-heading').find('.fa').removeClass('rotate');
+  })
+  .on('hidden.bs.modal', '#move-modal', function() {
+    $(this).remove();
+    foldersForMoving = {};
+  })
+  .on('change', '#move-model-organizations-select', function() {
+    $('.move-modal-empty-state').removeClass('active');
+    $('[data-move-button]').attr('disabled');
+    foldersForMoving['navStack'] = [];
+    var type = $(this).find(":selected").attr('data-type');
+    var id = $(this).find(":selected").attr('data-id');
+    var name = $(this).find(":selected").val();
+    addPathToStack(id, name, null, null, null, type);
+    searchFolders(id, type);
+  })
+  .on('click', '[data-move-folder]', function() {
+    if (!$(this).hasClass('children')) {
+      $('[data-move-folder]').removeClass('selected');
+      $(this).toggleClass('selected');
+    } else {
+      addPathToStack(
+        $(this).attr('data-folder-id'),
+        $(this).text(),
+        $(this).attr('data-org-id'),
+        $(this).attr('data-app-id'),
+        $(this).attr('data-parent-id'),
+        'folder');
```

Comment:
closing bracket should be on new line without indentation:

```
foo(
  1,
  2
);
```

Reply:
fixed